src/SystemTextJson.JsonDiffPatch.MSTest/SystemTextJson.JsonDiffPatch.MSTest.csproj

```diff
@@ -1,21 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <RootNamespace>System.Text.Json.JsonDiffPatch.MsTest</RootNamespace>
-  </PropertyGroup>
+    <PropertyGroup>
+        <RootNamespace>System.Text.Json.JsonDiffPatch.MsTest</RootNamespace>
+    </PropertyGroup>
 
-  <PropertyGroup>
-    <PackageId>SystemTextJson.JsonDiffPatch.MSTest</PackageId>
-    <PackageTags>mstest;mstest2;assert;compare;json;system-text-json</PackageTags>
-    <Description>Extension methods for MSTest v2 to compare JSON in unit tests.</Description>
-  </PropertyGroup>
+    <PropertyGroup>
+        <PackageId>SystemTextJson.JsonDiffPatch.MSTest</PackageId>
+        <PackageTags>mstest;mstest2;assert;compare;json;system-text-json</PackageTags>
+        <Description>Extension methods for MSTest v2 to compare JSON in unit tests.</Description>
+    </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\SystemTextJson.JsonDiffPatch\SystemTextJson.JsonDiffPatch.csproj" />
-  </ItemGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\SystemTextJson.JsonDiffPatch\SystemTextJson.JsonDiffPatch.csproj" />
+    </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
-  </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0">
```

Comment:
What's the purpose of adding this?

Reply:
I added SourceLink because it provides a great source debugging experience for end users. It provides a lot of debug-time assistance, allowing to step into the source code of the compiled NuGet package by pulling the source from GitHub, rather than trying to use decompilation. [SourceLink is supported by Microsoft and included in the .NET SDK](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink) for a variety of common source control providers, including GitHub, and adding it provides a quick win for a much improved package consumer experience.